src/pybind/pynmodl.cpp

```diff
@@ -301,6 +311,9 @@ PYBIND11_MODULE(_nmodl, m_nmodl) {
              "instance_size"_a,
              "cuda_grid_dim_x"_a = 1,
              "cuda_block_dim_x"_a = 1);
+#else
+        ;
```

Comment:
`pure nitpick`: only thing cached my attention in this PR is this! looked like pattern from legacy nrn code! 🙃

Reply:
I know it's definitely not nice :see_no_evil:
Do you have any suggestion about how this could be handled better?

Reply:
don't think so !